src/GraphQl/Resolver/Factory/ItemMutationResolverFactory.php

```diff
@@ -92,14 +92,21 @@ public function __invoke(string $resourceClass = null, string $rootClass = null,
             }
 
             $resourceMetadata = $this->resourceMetadataFactory->create($resourceClass);
-            $this->canAccess($this->resourceAccessChecker, $resourceMetadata, $resourceClass, $info, $item, $operationName);
+            if($operationName === 'update' || $operationName === 'delete') {
+                $this->canAccess($this->resourceAccessChecker, $resourceMetadata, $resourceClass, $info, $item, $operationName);
+            }
 
             switch ($operationName) {
                 case 'create':
                 case 'update':
                     $context = null === $item ? ['resource_class' => $resourceClass] : ['resource_class' => $resourceClass, 'object_to_populate' => $item];
                     $context += $resourceMetadata->getGraphqlAttribute($operationName, 'denormalization_context', [], true);
                     $item = $this->normalizer->denormalize($args['input'], $resourceClass, ItemNormalizer::FORMAT, $context);
+
+                    if($operationName === 'create') {
```

Comment:
Then it should be the same for `update` isn't it?

Reply:
I‘m not certain, because for update `$item` is already populated with the existing resource, so it is checked in the first condition. It depends on what kind of behaviour you want to allow.